powerauth-java-server/src/main/resources/application.properties

```diff
@@ -79,12 +79,16 @@ powerauth.service.recovery.maxFailedAttempts=5
 # Vault Unlock behavior
 powerauth.service.secureVault.enableBiometricAuthentication=false
 
+# Temporary Key Expiration in Milliseconds (default: 5 minutes)
```

Comment:
Here the milliseconds are ok because Spring can bind `30000` or `30s` to `Duration`.